scripts/queue.py

```diff
@@ -28,7 +28,7 @@
   -p, --pause SECONDS   Pause queues for a number of seconds. A value of 0
                         will unpause. If -m is passed, pause that queue,
                         otherwise pause all queues.
-""".format(archive_base=teuthology.config.config.archive_base)
+""".format(archive_base=teuthology.config.config.archive_base) # noqa
```

Comment:
the '{archive_base}' has been removed at some point, so there is no sense to use it, I guess there new flake8 version released and they raise severity of this warning, instead of `noqa` it is better to just remove `.format`

Reply:
yeah agree, i think the old flake8 missed those and its safe to remove the format where its not used.